src/screens/HomeScreen.js

```diff
@@ -159,193 +142,54 @@ export const HomeScreen = ({ navigation }) => {
-              <Fragment key={`${index}-${categoryTitle}`}>
-                <TitleContainer>
-                  <Touchable
-                    onPress={() =>
-                      navigation.navigate(
-                        NAVIGATION.NEWS_ITEMS_INDEX({
-                          categoryId,
-                          categoryTitle,
-                          categoryTitleDetail
-                        })
-                      )
-                    }
-                  >
-                    <Title accessibilityLabel={`${categoryTitle} (Überschrift) (Taste)`}>
-                      {categoryTitle}
-                    </Title>
-                  </Touchable>
-                </TitleContainer>
-                {device.platform === 'ios' && <TitleShadow />}
-                <Query
-                  query={getQuery(QUERY_TYPES.NEWS_ITEMS)}
-                  variables={{ limit: 3, ...{ categoryId } }}
-                  fetchPolicy={fetchPolicy}
-                >
-                  {({ data, loading }) => {
-                    if (loading) {
-                      return (
-                        <LoadingContainer>
-                          <ActivityIndicator color={colors.accent} />
-                        </LoadingContainer>
-                      );
-                    }
-
-                    const newsItems = parseNewsItems(
-                      data?.[QUERY_TYPES.NEWS_ITEMS],
-                      true,
-                      categoryTitleDetail
-                    );
-
-                    if (!newsItems || !newsItems.length) return null;
-
-                    const newsItemsListType = listTypesSettings[QUERY_TYPES.NEWS_ITEMS];
-
-                    return (
-                      <View>
-                        <ListComponent
-                          navigation={navigation}
-                          data={newsItems}
-                          query={QUERY_TYPES.NEWS_ITEMS}
-                          horizontal={newsItemsListType === LIST_TYPES.CARD_LIST}
-                        />
-
-                        <Wrapper>
-                          <Button
-                            title={categoryButton}
-                            onPress={() =>
-                              navigation.navigate(
-                                NAVIGATION.NEWS_ITEMS_INDEX({
-                                  categoryId,
-                                  categoryTitle,
-                                  categoryTitleDetail
-                                })
-                              )
-                            }
-                          />
-                        </Wrapper>
-                      </View>
-                    );
-                  }}
-                </Query>
-              </Fragment>
+            ({ categoryButton, categoryId, categoryTitle, categoryTitleDetail }, index) => (
+              <HomeSection
+                key={index}
```

Comment:
was war der grund für die `key` änderung? hab mal gelesen, dass nur ein index nicht optimal ist und glaube im gesamten bundle nicht eindeutig genug ist. auf die map hier bezogen is der zwar eindeutig, aber versuche extra immer noch was klar global eindeutigeres hinzuzufügen.

Reply:
https://de.reactjs.org/docs/lists-and-keys.html#keys

> Wir empfehlen dir nicht die Indizes für die Keys zu verwenden, da sich die Reihenfolge der Listeneinträge verändern kann. Dies kann sich negativ auf die Performance auswirken und zu Problemen mit dem Komponenten-State führen. Im Artikel von Robin Pokorny kannst du eine ausführlichere Erklärung über die negativen Auswirkungen beim Verwenden des Index als Key finden. Wenn du Listenelemente keinen expliziten Key zuweist, verwendet React standardmäßig die Indizes als Key.
>
> Wenn du dich dafür interessiert mehr zu lernen, ist hier eine ausführliche Erklärung, warum Keys nötig sind.

Reply:
I did not see the benefit of adding the category title to the key.
do you think there will be fluctuation in categories that might cause issues down the line?
should we check other occurences of the index being the key as well?

the keys only need to be unique between siblings, so i do not think there is any value in making them globally unique.